### ISSUE TITLE
font-family: -apple-system gives wrong warning

### ISSUE THREAD

**Issue by 1jj** [w3c/css-validator#80]
Validating http://elektronotdienst-nuernberg.de/bugs/prefix.html gives me this warning:

**Property -apple-system,system-ui,segoe ui,arimo,liberation sans,helvetica,sans-serif is an unknown vendor extension**

Thats of course wrong, because "-apple-system" is a font family and not a property

**Comment by sideshowbarker** [comment]
I can’t reproduce this with the following:
```css
body {
display : table;
font : -apple-system,system-ui,arimo,helvetica,sans-serif;
background : rgb(50, 80, 80);
}
```

…but I can with the following:

```css
body {
display : table;
font : -apple-system,system-ui,arimo,helvetica,sans-serif
background : rgb(50, 80, 80);
}
```

…that is, without the `;` at the end of the end of the line with the `font` declaration.

So I’m guessing that when you initially ran into this, it was due to some other syntax error (like a missing `;` or whatever), and that *Property -apple-system… is an unknown vendor extension* was a side effect of the parser getting tripped up because of that other syntax error.

Given that I’m closing this for now but if you can still reproduce this with a valid stylesheet, please post a comment and we can reopen it.
